src/components/SearchBar.jsx

```diff
@@ -229,11 +229,13 @@ class SearchBar extends Component {
     this.setState({ query: null, searching: false })
   }
 
-  onSuggestionSelected = (event, { suggestion }) => {
+  onSuggestionSelected = async (event, { suggestion }) => {
     const { onSelect } = suggestion
     // `onSelect` is a string that describes what should happen when the suggestion is selected. Currently, the only format we're supporting is `open:http://example.com` to change the url of the current page.
 
-    if (/^open:/.test(onSelect)) {
+    if (typeof onSelect === 'function') {
+      await onSelect()
```

Comment:
Is it 100% safe to have this await without a try catch here?

Reply:
https://github.com/cozy/cozy-drive/pull/2663/files#r942584320